Scenes/Kaleidoscope/KaleidoscopeViewport.tscn

```diff
@@ -5,3 +5,5 @@ anchor_right = 1.0
 anchor_bottom = 1.0
 
 [node name="Viewport" type="Viewport" parent="."]
+handle_input_locally = false
+render_target_update_mode = 3
```

Comment:
I'm not sure how these changes relate to the sound button? I'd prefer to remove them from the merge request if they are unrelated.

Reply:
i didn't change anything in this, but i have opened the file to see what it looks like, maybe my godot added this ?
i am using 3.4.4

Reply:
Yeah, probably it just added it. I'll switch to 3.4.4 (from 3.5-rc6) and reload everything before I cause anyone further unexpected issues.

Reply:
ok, i can see there is a request change, should i add a new commit with these 2 lines removed ?